test/cp-cli.test.ts

```diff
@@ -29,16 +30,18 @@ describe('cp-cli', () => {
     });
   });
 
-  it('should dereference symlinks', async () => {
-    shell.cd('test/assets');
-    shell.ln('-s', 'foo.txt', 'bar.txt');
-    shell.cd('../..');
-    const { stderr } = shell.exec('node bin/cp-cli -d test/assets out');
-    expect(stderr).toEqual('');
-    let stats = await fse.stat('out/foo.txt');
-    expect(stats.isFile()).toEqual(true);
-    stats = await fse.stat('out/bar.txt');
-    expect(stats.isFile()).toEqual(true);
-    expect(stats.isSymbolicLink()).toEqual(false);
-  });
+  if (platform() !== 'win32') {
```

Comment:
It's too bad jest doesn't have `mocha`'s `skip()`.

Reply:
You [can](https://facebook.github.io/jest/docs/en/api.html#testskipname-fn) 😉

Reply:
Jest's `.skip` only supports static skipping. If you look at `mocha's` (.skip)[https://mochajs.org/#inclusive-tests] it lets you conditionally call it. The way this code is now, the test is silently omitted on windows (which isn't ideal).

Reply:
Ah I understand. That's a cool feature and I didn't knew that. Maybe [skip-if](https://github.com/relekang/skip-if) is what you are looking for :wink: Unfortunately it's not build in into Jest: https://github.com/facebook/jest/issues/3652

Reply:
Yeah, I didn't want to add a dependency to your module just for this. Thanks for releasing a new version!